Fixtures/BinaryLibraries/Static/Package1/Simple.artifactbundle/info.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
suggestion: This file does not contain support for Windows, and the associated test does not skip this test on Windows.

I suspect we will either (1) need to add Support for windows, or (2) skip this on windows

If we need to support this feature for Windows and we decide to skip the test on Windows, please create a GitHub issue and reference the issue in the comment with skipping the test.  We can use `XCTSkipOnWindows(...)` in SwiftPM's `_InternalTestSupport` module for ths.